package.json

```diff
@@ -99,37 +111,27 @@
     "tslint-no-circular-imports": "0.5.0",
     "typescript": "2.8.4"
   },
+  "engines": {
+    "node": ">=8.0.0 ",
+    "npm": ">=5.0.0 ",
+    "yarn": ">=1.0.0 "
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "homepage-comment": [
```

Comment:
This was intended to go above "homepage", if not there, is not worth to have it at all.

Reply:
This is the generated order by the sort-package-json script.
I'm fine moving the comments to a more logical place though.